contracts/WildcardSteward_v3.sol

```diff
@@ -72,6 +65,14 @@ contract WildcardSteward_v3 is Initializable {
     mapping(address => uint256) public benefactorCredit;
     address public withdrawCheckerAdmin;
 
+    /*
+    31536000 seconds = 365 days
+
+    divisor = 365 days * 1000000000000
+            = 31536000000000000000
+    */
+    uint256 constant yearTimePatronagDenominator = 31536000000000000000;
```

Comment:
I made this a global constant. The `constant` keyword ensures there is no extra gas cost.

Reply:
excellent